src/openrct2/interface/Window.h

```diff
@@ -137,6 +137,18 @@ struct Widget
         return top - 1;
     }
 
+    void moveRight(int32_t amount)
```

Comment:
This one is unused. Is it intentional?

Reply:
It is, yes - I added a function to move down (and up), so I figured a similar function for right (and left) would be nice as well. I can remove it if desired, though.

Reply:
It's ok to leave it. I dont see moveUp, but I guess it's just a matter of moving down by a negative value?